app/jobs/spending_breakdown_job.rb

```diff
@@ -33,7 +33,7 @@ def save_tempfile(export)
   end
 
   def upload_csv_to_s3(file:, filename:)
-    Export::S3Uploader.new(file: file, filename: filename, use_public_bucket: true).upload
+    Export::S3Uploader.new(file: file, filename: filename, use_public_bucket: false).upload
```

Comment:
Can we remove `use_public_bucket` altogether now, given we're moving to always private for everything?

Reply:
We should be able to - maybe we can do that as a separate PR though to keep this one small?